### PR TITLE
AdminSessionsPageUiTest

### PR DESCRIPTION
**Pull request by JosephineKwa**
Used wild cards for the filter dates, table contents and table count.

### CODE REVIEW

**Comment by JosephineKwa** [comment]
Ready for review :)

File: src/test/java/teammates/test/cases/ui/browsertests/AdminSessionsPageUiTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    private void testContent() {
+        
+        ______TS("content: typical page");
+        
+        Url sessionsUrl = createUrl(Const.ActionURIs.ADMIN_SESSIONS_PAGE);
+        sessionsPage = loginAdminToPage(browser, sessionsUrl, AdminSessionsPage.class);
+        sessionsPage.verifyHtml("/adminSessionsPage.html");
+        
+        ______TS("content: show filter");
+
```
Comment (taniach) [inline src/test/java/teammates/test/cases/ui/browsertests/AdminSessionsPageUiTest.java]: Add __TS("content: hide filter") also

**Comment by taniach** [comment]
@JosephineKwa some changes requested

**Comment by JosephineKwa** [comment]
Ready for review again :D

**Comment by taniach** [comment]
@JosephineKwa good to merge :)

**Pull request merged**